src/Provider/URLBuilder.js

```diff
@@ -71,10 +71,10 @@ export default {
      */
     bbox: function bbox(bbox, layer) {
         const box = bbox.as(layer.projection);
-        const w = box.west();
-        const s = box.south();
-        const e = box.east();
-        const n = box.north();
+        const w = box.west().toFixed(5);
```

Comment:
I'm not sure there's enough number after the comma

Reply:
For layers using metres this won't be a problem (and we could reduce to `2`).

For 4326, `5` might be a bit low, so I can increase it to 9.

Reply:
I've updated the branch.

Reply:
> For 4326, 5 might be a bit low, so I can increase it to 9.

yes, `1e-9` is enough because the projection on the globe is less than 1 cm